composer/core/logging/base_backend.py

```diff
@@ -45,8 +49,7 @@ def log_metric(self, timestamp: Timestamp, log_level: LogLevel, data: TLogData):
         (e.g. write it to a file, send it to a server, etc...).
 
         Args:
-            epoch (int): The epoch for the logged data.
-            step (int): The global step for the logged data.
+            timestamp (Timestamp): The timestamp for the logged data.
             log_level (LogLevel): The log level.
             data (TLogData): The metric to log.
```

Comment:
TLogData didn't quite resolve by sphinx here: https://mosaicml-composer--499.com.readthedocs.build/en/499/api_reference/composer.core.logging.base_backend.html#composer.core.logging.base_backend.LoggerCallback

This one is actually quite important since its the type of the data, so we should find a way to link it.

Reply:
I haven't been able to solve this. Perhaps @ravi-mosaicml or @bandish-shah might be able to help 🙏